emcc.py

```diff
@@ -1524,9 +1524,6 @@ def setup_pthreads(target):
   # set location of worker.js
   settings.PTHREAD_WORKER_FILE = unsuffixed_basename(target) + '.worker.js'
 
-  # memalign is used to ensure allocated thread stacks are aligned.
-  settings.REQUIRED_EXPORTS += ['emscripten_builtin_memalign']
```

Comment:
The `settings.REQUIRED_EXPORTS` setting is an internal setting that requires that function be exported from the wasm to the JS module code.   It doesn't mean it exported outside the module, so this would only be user visible if the user have module-internal JS code that called `emscripten_builtin_memalign` without declaring their dependency on it some way. 

So I don't' think its worth a changelog entry.